cloud/infoblox

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/python
+#
+# Copyright 2014 "Igor Feoktistov" <ifeoktistov@yahoo.com>
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import re
+import requests
+import json
+import os
+import urllib3
```

Comment:
Since urllib is potentially not installed on the remote system, you should check for the module and fail if it not here Ditto for request, I guess.